app/src/main/java/chat/rocket/android/app/chatroom/MessageListAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package chat.rocket.android.app.chatroom
+
+import DateTimeHelper
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import chat.rocket.android.R
+import com.facebook.drawee.view.SimpleDraweeView
+import kotlinx.android.synthetic.main.item_message.view.*
+
+class MessageListAdapter(private var dataSet: MutableList<Message>) : RecyclerView.Adapter<MessageListAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.item_message, parent, false)
```

Comment:
We should lazy initialize a `inflater` field instead of calling  `LayoutInflater.from(parent.context)` on every `onCreateViewHolder()` call